src/context-menu-builder.js

```diff
@@ -156,9 +156,12 @@ export default class ContextMenuBuilder {
     let copyLink = new MenuItem({
       label: isEmailAddress ? this.stringTable.copyMail() : this.stringTable.copyLinkUrl(),
       click: () => {
-        // Omit the mailto: portion of the link; we just want the address
-        clipboard.writeText(isEmailAddress ?
-          menuInfo.linkText : menuInfo.linkURL);
+        let url = menuInfo.linkURL;
+        if (isEmailAddress) {
+          // Omit the mailto: portion of the link; we just want the address
+          url = url.slice('mailto:'.length);
```

Comment:
We should also strip anything after `?` if present, because it's not useful in the clipboard.

Reply:
Email addresses are technically allowed to have question marks. Was there a specific case you were thinking about?

Reply:
I was thinking of `mailto` links that have `?subject=foo&body=bar`.

Reply:
I'm concerned whether that will always be the correct behavior. If a mailto link has those attributes, and it's copied, the user can remove it. If we remove it for them they can't add it back.

Reply:
The menu says "Copy Email Address". I think it should just copy an email address.

My guess is that most people want just the email address, and would be surprised to see other stuff. I think we should optimize for that.

(If you copy `tim@superhuman.com?subject=foo` and paste in a compose form, it looks silly and weird)

Reply:
That's what Chrome does actually.